README.md

```diff
@@ -1,26 +1,159 @@
-# ember-exclaim
+# ember-exclaim [![Build Status](https://travis-ci.org/salsify/ember-exclaim.svg?branch=master)](https://travis-ci.org/salsify/ember-exclaim)
 
-This README outlines the details of collaborating on this Ember addon.
+This addon allows apps to expose declarative, JSON-configurable custom UIs that are backed by Ember components.
 
-## Installation
+## Simple Examples
 
-* `git clone <repository-url>` this repository
-* `cd ember-exclaim`
-* `npm install`
+The building blocks available to an ember-exclaim UI are defined by the app it's used in, but the dummy application in this project contains implementations of several basic components that might be useful. For example, in the [demo application](https://salsify.github.io/ember-exclaim), the following would render a header with some filler content below it:
 
-## Running
+```js
+{
+  "$vbox": [
+    { "$header": "Very Important Content" },
+    { "$text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua." }
+  ]
+}
+```
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+<img src="https://user-images.githubusercontent.com/108688/27345599-4e40b902-55b8-11e7-8e65-195117aeded4.png" width="369">
 
-## Running Tests
+Components can also read from and write back to an underlying data structure. For example, given this environment:
 
-* `npm test` (Runs `ember try:each` to test your addon against multiple Ember versions)
-* `ember test`
-* `ember test --server`
+```js
+{
+  "target": "world"
+}
+```
 
-## Building
+The following UI would render "Hello, world!":
 
-* `ember build`
+```js
+{
+  "$text": ["Hello, ", { "$bind": "target" }, "!"]
+}
+```
 
-For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
+And something like this would render an input that would update the underlying value of `target` as the user made changes:
+
+```json
```

Comment:
Nice catch — I played with both and didn't love how Github highlighted either. Will fix